docker/Dockerfile

```
@@ -31,31 +31,26 @@ RUN npm run build
 FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION} AS backend-build
 
 # git is needed to install streaming-form-data fork
-# libffi-dev is needed to fix poetry dependencies for >= v1.8 on arm64
 # libpq-dev is needed to build psycopg-c
 # mariadb-connector-c-dev is needed to build mariadb-connector
 RUN apk add --no-cache \
     gcc \
     git \
-    libffi-dev \
     libpq-dev \
     mariadb-connector-c-dev \
     musl-dev
 
-RUN pip install poetry
-
-ENV POETRY_NO_INTERACTION=1 \
-    POETRY_VIRTUALENVS_IN_PROJECT=1
+RUN curl -LsSf https://astral.sh/uv/install.sh | env UV_INSTALL_DIR="/usr/local/bin" sh
```

We could install `uv` by copying the executables from their Docker image: https://docs.astral.sh/uv/guides/integration/docker/#installing-uv

```suggestion
COPY --from=ghcr.io/astral-sh/uv:0.7.19 /uv /uvx /bin/
```